docs/source/form_boundary_conditions.rst.bak

```diff
@@ -0,0 +1,191 @@
+.. default-role:: math
```

Comment:
Don't commit backups and other generated files.

Reply:
This looks like a consequence of using `git add -A`, it is best to avoid doing this, and instead specifically say `git add` on each file that you want to add to a commit. These days, if you use